literature.bib

```diff
@@ -7820,6 +7885,43 @@ @inproceedings{RAK+:VaMoS13
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2012 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@article{H:JSS12,
+	author = {Geir Kjetil Hanssen},
+	title = {{A Longitudinal Case Study of an Emerging Software Ecosystem: Implications for Practice and Theory}},
+	journal = JSS,
+	volume = {85},
+	number = {7},
+	pages = {1455--1466},
+	url = {https://doi.org/10.1016/j.jss.2011.04.020},
+	doi = {10.1016/j.jss.2011.04.020},
+	year = 2012
+}
+
+@inproceedings{HD:IWSECO12,
+	author = {Geir Kjetil Hanssen and Tore Dyb{\aa}},
+	title = {{Theoretical Foundations of Software Ecosystems}},
+	booktitle = IWSECO,
+	location = {Cambridge, MA, USA},
+	series = {{CEUR} Workshop Proceedings},
+	volume = {879},
+	pages = {6--17},
+	publisher = CEUR-WS,
+	address = Aachen,
+	url = {http://ceur-ws.org/Vol-879/paper1.pdf},
+	year = 2012
+}
+
+@article{BWB:BISE12,
+	title = {{Software Ecosystems}},
+	author = {Burkard, Christoph and Widjaja, Thomas and Buxmann, Peter},
+	journal = BISE,
+	volume = {4},
+	number = {1},
+	pages = {41--44},
+	publisher = Springer,
```

Comment:
Address missing, especially in case of Springer, this is an important piece of information.

Reply:
won't fix because journal as above; also I was not able to find the address